mappings/net/minecraft/block/DecoratedPotPatterns.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/unmapped/C_hzdaoazh net/minecraft/block/DecoratedPotPatterns
+	FIELD f_auhlbqrf POTTY_PATTERN_SKULL_NAME Ljava/lang/String;
+	FIELD f_cpwmrhfm DECORATED_POT_SIDE_NAME Ljava/lang/String;
+	FIELD f_eomvlzyj POTTERY_PATTERN_ARMS_UP_NAME Ljava/lang/String;
+	FIELD f_iuwqdcay DECORATED_POT_BASE_NAME Ljava/lang/String;
+	FIELD f_jpiddhhm POTTERY_PATTERN_ARCHER_NAME Ljava/lang/String;
+	FIELD f_pkicorcb POTTERY_PATTERN_PRIZE_NAME Ljava/lang/String;
+	FIELD f_tssagzls SHARD_TO_PATTERN Ljava/util/Map;
```

Comment:
hm, if i assume this is what it is, shouldn't it be:
```suggestion
	FIELD f_tssagzls PATTERNS_BY_SHARDS Ljava/util/Map;
```
in order to fit conventions?